scripts/meterpreter/multi_console_command.rb

```diff
@@ -25,18 +26,28 @@
 commands = nil
 script = []
 help = 0
+silence = 0
 
 ################## Function Declarations ##################
 # Function for running a list of commands stored in a array, returs string
```

Comment:
Not your code, but: # Function for running a list of commands stored in a array, returns string